src/FixedDataTableBufferedRows.js

```diff
@@ -94,6 +95,7 @@ class FixedDataTableBufferedRows extends React.Component {
             scrollLeft={Math.round(props.scrollLeft)}
             visible={false}
             fixedColumns={props.fixedColumns}
+            fixedRightColumns={props.fixedColumns}
```

Comment:
This should have been done as part of #375.
Since `visible` is falsy, it doesn't matter if we pass `fixedRightColumns` or not. **But** it does remove the warning made by React's prop type checking.

Reply:
Thanks for fixing.  

Do we need to pass onContextMenu as well?  Or is that not required for the row?

Reply:
Since these rows are rendered as null, there's no way `onContextMenu` can be called. And since the prop is not _required_, it's omitted from passing.